src/sif/nocost.cc

```diff
@@ -192,7 +192,9 @@ class NoCost : public DynamicCost {
    * mode used by the costing method. It's also used to filter
    * edges not usable / inaccessible by automobile.
    */
-  float Filter(const baldr::DirectedEdge* edge, const graph_tile_ptr&) const override {
+  bool Allowed(const baldr::DirectedEdge* edge,
+               const graph_tile_ptr&,
+               uint16_t disallow_mask = kDisallowNone) const override {
     return !(edge->is_shortcut() || edge->IsTransitLine());
```

Comment:
the point of this costing is that it doesnt need to be conservative which is why its Implementation didnt change